src/as2fm/scxml_converter/scxml_entries/scxml_param.py

```diff
@@ -42,8 +42,8 @@ class ScxmlParam(ScxmlBase):
     def get_tag_name() -> str:
         return "param"
 
-    @staticmethod
-    def from_xml_tree(xml_tree: ET.Element) -> "ScxmlParam":
+    @classmethod
```

Comment:
As before, does it make sense to use the `classmethod` decorator for static methods not accessing the actual `cls` value?

Reply:
I think it is not possible to mix these. The problem is, that some `from_xml_tree_impl` definitely need to be classmethods, for example the one in `RosDeclaration`. And then all have to be classmethods, because the signatures are different.